packages/main/test/pages/Avatar.html

```diff
@@ -151,6 +151,15 @@ <h3>Avatar - test</h3>
 		<ui5-avatar id="myAvatar5" initials="WWW" shape="Square" size="M"></ui5-avatar>
 	</section>
 
+	<section>
+		<h3>Avatar - interactive disabled</h3>
+		<ui5-avatar id="interactive-disabled" disabled aria-haspopup="menu" interactive initials="XS" size="XS" accessible-name="Interactive avatar"></ui5-avatar>
```

Comment:
do we need aria-haspopup attribute?

Reply:
This is a part of the test code so it's irrelevant to the end user.